batch-setup/make_meta_tiles.py

```diff
@@ -457,15 +494,36 @@ def render(self, num_retries, lense):
     assert args.metatile_size > 0
     assert args.metatile_size < 100
     metatile_max_zoom = 16 - metatile_zoom_from_size(args.metatile_size)
+
+    bbox_west = -181
```

Comment:
> since either you are setting `use_tiles_coords_generator` to True and must provide all the sides, you probably don't need these defaults

Yeah, but this is just to avoid IDE linter warnings such as "bbox_south might be None"

Reply:
Hmmm.  I don't love that we're including this code to prevent a thing we know can't happen.  I guess do whatever you want there, but this feels questionable

Reply:
removed in the newest commit